CMakeLists.txt

```
@@ -381,6 +381,7 @@ if(${VELOX_ENABLE_GPU})
     add_compile_options("$<$<COMPILE_LANGUAGE:CUDA>:-G>")
   endif()
   include_directories("${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES}")
+  find_package(CUDAToolkit REQUIRED)
```

```suggestion
    include(FindCUDAToolkit)
```
See https://cmake.org/cmake/help/v3.23/module/FindCUDAToolkit.html#imported-targets

---

Hm... it's found which should then also create the targets. 
After some local investigation (in the container) it seems we need to install the nvrtc library explicitly but even then cmake fails to find the library location for some reason. In a standalone cml.txt without anything but the include the target is not created because the lib is not found:
>   CUDA_nvrtc_LIBRARY-NOTFOUND

even if the lib is clearly on the system:
> /usr/local/cuda-12.4/targets/x86_64-linux/lib/libnvrtc.so.12

and the path is correct in the find command:
> find_library(CUDA_nvrtc_LIBRARY NAMES nvrtc HINTS /usr/local/cuda-12.4/targets/x86_64-linux/lib;/usr/local/cuda/targets/x86_64-linux/lib/stubs;/usr/local/cuda/targets/x86_64-linux/lib ENV CUDA_PATH PATH_SUFFIXES lib64/stubs lib/x64/stubs lib/stubs stubs )

---

On my own machine the same script works fine, used the same current cmake in the container as well but it still fails the same way.